.gitignore

```diff
@@ -8,3 +8,8 @@ dist/
 
 # Jetbrains
 .idea/
+
```

Comment:
includes https://github.com/mattermost/mattermost-plugin-apps/pull/84 for debugging